microsoft-edge/devtools-guide-chromium/javascript/index.md

```diff
@@ -182,7 +182,7 @@ The **Watch** pane allows you to monitor the values of variables (such as `sum`)
 1. Press `Enter`.  DevTools displays `typeof sum: "string"`.  The value to the right of the colon is the result of your Watch Expression.
```

Comment:
Dan wrote on PR main page:
line 178. unable to add watch expression @JasonAndrewWriter maybe Patrick knows someone who can follow up on this issue

Reply:
thanks

Reply:
pls close this comment.
Latest: 
https://github.com/MicrosoftDocs/edge-developer/pull/2309/files